sqld/src/query_analysis.rs

```diff
@@ -200,25 +200,44 @@ impl Statement {
                 is_insert,
             })
         }
+
+        // Workaround for CREATE FUNCTION.
+        // FIXME: this should be handled by the parser (or *at least* trimmed + case insensitive)
+        let is_create_function = s.starts_with("CREATE FUNCTION");
```

Comment:
yeah this needs to be handled by the parser 100%, trimming and case insensitive won't be enough, there can be an arbitrary number of whitespace between `create` and `function`